src/config.js

```diff
@@ -14,9 +15,17 @@ const Config = CoreObject.extend({
    * @param {String} attrs.appId The app whose listings the client will be
    * using. If you are just modifying a buy button, the buy-button's app id is
    * 6. Otherwise, obtain the app id of the app you're modifying or extending.
-   * @param {String} attrs.myShopifyDomain You shop's `myshopify.com` domain.
+   * @param {String} attrs.domain Your shop's full domain.
```

Comment:
It would be good to add an example here, so people don't associate it with the domains they add to their shop (if they're using Online Store). 
```diff
-   * @param {String} attrs.domain Your shop's full domain.
+   * @param {String} attrs.domain Your shop's full domain (i.e. `craftedgoods.myshopify.com`)
```